sdk/python/feast/cli.py

```diff
@@ -753,6 +756,22 @@ def serve_transformations_command(ctx: click.Context, port: int):
     store.serve_transformations(port)
 
 
+@cli.command("serve_registry")
+@click.option(
+    "--port",
+    "-p",
+    type=click.INT,
+    default=DEFAULT_REGISTRY_SERVER_PORT,
+    help="Specify a port for the server",
+)
+@click.pass_context
+def serve_registry_command(ctx: click.Context, port: int):
+    """Start a registry server locally on a given port."""
+    store = create_feature_store(ctx)
+
+    store.serve_registry(port)
```

Comment:
I expect there should be an option to broadcast the TTL cache to downstream function as well

Reply:
Do you mean cache_ttl_seconds config? That's not part of BaseRegistry interface and one would be able to configure it in feature_store.yaml anyway, right?

Reply:
grpc registry server should have a cache itself to prevent concurrent repeated calls to the server

I believe we should add as an option, it's user's decision to keep it or not:

- default ttl may be considered as a small number (less than or equal 60 secs) 
- to disable cache, set it to -1

Reply:
The cache should be an option, I agree, but all registry implementations do have client-side (In this case the registry server itself acts as a "client") caches already that will accomplish exactly what you are looking for, no? If we add a separate cache for a registry server, you would end up having two identical chained caches right next to each other in the same python process. What would be an added benefit? Plus, it will be technically possible to specify different ttls for them and that's bound to get confusing.

Reply:
yeah got it, think we should avoid confusion as you mentioned 🙌 
everything else looks good to me